.sonarcloud.properties

```diff
@@ -0,0 +1,15 @@
+# Path to sources
+#sonar.sources=.
+#sonar.exclusions=
+#sonar.inclusions=
```

Comment:
This file is unnecessary, you can delete it

Reply:
removed